app/controllers/PdfGenerationController.scala

```diff
@@ -59,8 +60,16 @@ trait PdfGenerationController extends ERSReturnBaseController with Authenticator
           if (all.getEntry[ReportableEvents](CacheUtil.reportableEvents).get.isNilReturn.get == PageBuilder.OPTION_UPLOAD_SPREEDSHEET) {
             val fileType = all.getEntry[CheckFileType](CacheUtil.FILE_TYPE_CACHE).get.checkFileType.get
             if (fileType == PageBuilder.OPTION_CSV) {
-              val csvFilesCallback: List[CsvFilesCallback] = all.getEntry[CsvFilesCallbackList](CacheUtil.CHECK_CSV_FILES).get.files
-              for (file <- csvFilesCallback if file.callbackData.isDefined) {
+              val csvCallback = all.getEntry[UpscanCsvFilesCallbackList](CacheUtil.CHECK_CSV_FILES).get
```

Comment:
Use of .get here.

Reply:
Will change the exception thrown here as an exception is the desired behaviour